lib/whitehall_importer/create_edition.rb

```diff
@@ -194,5 +196,66 @@ def create_revision_history(edition)
         )
       end
     end
+
+    def create_notes(edition)
+      whitehall_edition["editorial_remarks"].each do |event|
+        details = TimelineEntry::WhitehallImportedEntry.create!(
+          entry_type: "internal_note",
+          body: {
+            "body": event["body"],
+          }.to_json,
+        )
+        TimelineEntry.create!(
```

Comment:
Since we've got a few instances of creating a TimelineEntry, perhaps it would be worth DRYing it up by extracting it out into a `whitehall_migration_timeline_entry` (or similar) method to which we pass the edition, `created_at`, `created_by` and `details`.

Reply:
@emmabeynon Thanks for the comments.  Will pull this and the `details = TimelineEntry::WhitehallImportedEntry...` into methods and DRY up.